src/hp_auth.erl

```diff
@@ -56,3 +57,12 @@ reset_password(Email) ->
   VerificationCode = base64url:encode(crypto:strong_rand_bytes(20)),
   db_user:reset_password(Email, VerificationCode),
   hp_email:send_password_reset(Email, VerificationCode).
+
+build_holiday_access_token(Email, Name, AvatarUrl) ->
+  Data = #{
+    email => Email,
+    name => Name,
+    avatar => AvatarUrl
+   },
+  {ok, Token} = hp_auth:token_encode(Data),
```

Comment:
```suggestion
  {ok, Token} = token_encode(Data),
```

Reply:
I suspect you could arrange things so token_encode is not exported anymore (not sure)

Reply:
The only other function that calls `token_encode` is `to_json/2` in `token_handler.erl`, but I don't believe it could be easily moved